src/NexusMods.App.UI/Overlays/Login/StartupMessageBox/LoginMessageBoxViewModel.cs

```diff
@@ -32,6 +32,11 @@ public LoginMessageBoxViewModel(ISettingsManager settingsManager, ILoginManager
     
     public bool MaybeShow()
     {
+        // If we're already logged in via OAuth, don't show the login modal
+        if (_loginManager.IsOAuthLogin) 
+            return false;
+        
+        // If we've already shown the modal, don't show it again
         if (_settingsManager.Get<LoginSettings>().HasShownModal) return false;
```

Comment:
`HasShownModal` ended up being false in release builds here?

Reply:
That seems really odd. The point of this method is to check whether the user has already seen the modal or not. The login status doesn't matter. I feel like this change doesn't solve the underlying issue, whatever that is.